internal/pkg/controllers/spod/bindata/spod.go

```diff
@@ -39,6 +40,7 @@ const (
 	SelinuxDropDirectory = "/etc/selinux.d"
 	SelinuxdSocketDir    = "/var/run/selinuxd"
 	SelinuxdSocketPath   = SelinuxdSocketDir + "/selinuxd.sock"
+	SyslogDropDirectory  = "/var/log/syslog"
```

Comment:
So it looks like that we have a direct dependency to `syslogd` (and/or auditd?). Not all distributions ship it per default and stick to journald. I think we can work with this limitation, I'm just wondering how to make it accessible to the broadest possible mass.

Reply:
`/var/log/syslog` isn't avialable in CoreOS. IIRC in some distros, syslogd isn't enabled either but these logs come from a journald plugin that persists the log there. We can keep this for now, but at some point we need to make this configurable or pluggable. I guess we'd need a journald driver for this to work on other distros.

Reply:
I will make a few tests using `/dev/kmsg` instead. But would be interesting to know its support across distros.